### ISSUE TITLE
pretrained weights

### ISSUE THREAD

**Issue by mangye16** [antspy/inception_v1.pytorch#5]
Hi, I have evaluated the code with another image normalization parameters, the performance is much better. But still, I'm not sure whether it's correct or not. Could u pls tell me where the pretrained weights come from?

**Comment by antspy** [comment]
Hi, 

the pretrained weights come from the repo linked in the README file (https://github.com/soumith/inception.torch ). If you find a mistake in my image normalization code, please don't hesitate to issue a pull request (or just tell me here). Thank you!

**Comment by mangye16** [comment]
Actually, I use the default normalization parameter of pytorch, the performance is much better for my task. I'm not sure whether it works for the ImageNet testing. You may try it.
normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406],std=[0.229, 0.224, 0.225])

**Comment by antspy** [comment]
Thanks! What is the accuracy that you get with these settings?

**Comment by mangye16** [comment]
Actually, I only test it on CUB200 dataset using the pre-trained weights. 
Using your normalization parameter the accuracy is:  recall @1 =26.5%, nmi =38.2%
Using the above-mentioned parameter the accuracy is:  recall @1 =39.1%, nmi =49.5%
I also tried normalize = transforms.Normalize(mean=[0.4588, 0.4588, 0.4588],std=[0.2, 0.2, 0.2]), the accuracy is: recall @1 =40.1%, nmi =49.8%

However, the pretrained inception v1 model with Tensorflow on CUB200 dataset is about recall @1 =43%, nmi = 49%.

Therefore, I'm not sure which parameter is correct. Could u pls check it on ImageNet? Thanks a lot.
